# ARGs-OAP-v2.0
This is the source code for ARGs-OAP v2.0 construction

Copyright: LG209, Environmental Microbiome Engineering and Biotechnology Laboratory, The University of Hong Kong

Author   : Xiaole Yin, email: sdulele@163.com

Citation : Yin, X, X-T Jiang, B. Chai, L. Li, Y. Yang, J. R. Cole, J. M. Tiedje*, and T. Zhang*.(2018) "ARGs-OAP v2. 0 with an Expanded SARG Database and Hidden Markov Models for Enhancement Characterization and Quantification of Antibiotic Resistance Genes in Environmental Metagenomes." Bioinformatics 1:8. DOI:10.1093/bioinformatics/bty053

To analyze metagenomic datasets through ARGs-OAP v2.0, please visit http://smile.hku.hk/SARGs

There are two modules in ARGs-OAP v2.0. One is for alignment of reads against SARG v2.0 database through first-stage UBLAST and second-stage BLAST to identify and quantify ARGs. The second modules is based on Sargfam, a set of profile Hidden Markov Models of ARGs, to annotate ARGs-like protein sequences (eg. ORFs).
